File: javascript/8kyu/get-planet-name-by-id.js
/*

Description:
The function is not returning the correct values. Can you figure out why?

getPlanetName(3); // should return 'Earth'

*/

function getPlanetName(id) {
  const planets = [
    "Mercury",
    "Venus",
    "Earth",
    "Mars",
    "Jupiter",
    "Saturn",
    "Uranus",
    "Neptune",
  ];
  return planets[id - 1];
}

File: cpp/8kyu/get_planet_name_by_id.cpp
/*

Author: MARCEL EBALLAR
GitHub: https://github.com/MACLEB1903

Source: 
https://www.codewars.com/kata/515e188a311df01cba000003/train/cpp


Instructions:
The function is not returning the correct values. Can you figure out why?

Example (Input --> Output ):
3 --> "Earth"

*/

#include <string>
using namespace std;

string get_planet_name(int id){
    switch (id) {
    case 1: return "Mercury";
    case 2: return "Venus";
    case 3: return "Earth";
    case 4: return "Mars";
    case 5: return "Jupiter";
    case 6: return "Saturn";
    case 7: return "Uranus";
    case 8: return "Neptune";
    }
}